lib/reline/line_editor.rb

```diff
@@ -1645,7 +1645,7 @@ def call_completion_proc_with_checking_args(pre, target, post)
       @line = ' ' * new_indent + @line.lstrip
 
       new_indent = nil
-      result = @auto_indent_proc.(new_lines[0..-2], @line_index - 1, (new_lines[-2].size + 1), false)
+      result = @auto_indent_proc.(new_lines[0..-2], @line_index - 1, (new_lines[@line_index - 1].bytesize + 1), false)
```

Comment:
Third arg is a byte_pointer, so it should use `bytesize` instead of `size`

I think the intention of this line is to pass `line_number` and `new_lines[line_number].bytesize + 1`.
But `new_lines[-2]` was not always `new_lines[@line_index - 1]`. Fixed the index.

Reply:
Thanks for the explanation! Would you mind extracting these arguments into local variables so they'd be self-documenting?

Reply:
I see. Let's keep it as it is then 👍